src/main/resources/library/attribute-definitions.json

```diff
@@ -43,9 +46,18 @@
       "items": { "type": "string" },
       "title": "Research Institute"
     },
+    "library:diseaseOntologyDOID": {
+      "type": "string",
+      "hidden": true
+    },
+    "library:diseaseOntology": {
+      "type": "string",
+      "title": "Cohort Phenotypic Inclusion Criteria (Disease Ontology)",
+      "typeahead": "ontology"
+    },
```

Comment:
My one request towards generic-ness: rename these attrs to "fooId" and "fooLabel" ... so, `library:diseaseOntologyId` and `library:diseaseOntologyLabel`. While we don't need to make our code totally generic, it would be a pain to have to update attributes already saved onto workspaces.

By using a naming convention of "fooId" and "fooLabel", we could write code in the future that programmatically generates the names. So, if we added a new ontology-driven property for, say, the research area, we could know to expect `library:researchAreaId" and `library:researchAreaLabel`.

Unfortunately, renaming these here will require code changes in the UI.